packages/mongoose-store/src/config.ts

```diff
@@ -0,0 +1,24 @@
+// TODO move to modules to decide what datastore they should use.
+export default {
```

Comment:
:+1: for this, but I'm going to defer this change to https://issues.jboss.org/browse/RAINCATCH-844

In this issue I was mostly interested in seeing what are the steps needed to migrate one of our existing leaf modules over. We can avoid merging this if we settle on another approach for the store